examples/slack/README.md

```diff
@@ -0,0 +1,195 @@
+# Slack example
+
+This `repo.yaml` file contains a workflow specification to send Infracost cost estimates to Slack.
+
+Slack message blocks have a 3000 char limit so the Infracost CLI automatically truncates the middle of `slack-message` output format.
+
+## Table of Contents
+
+* [Running with GitHub](#running-with-github)
+* [Running with GitLab](#running-with-gitlab)
+* [Running with Azure Repos](#running-with-azure-repos)
+
+## Running with GitHub
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) Docker image.
+2. You'll need to pass the following custom env vars into the container:
+   ```sh
+   GITHUB_TOKEN=<your-github-token>
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   SLACK_WEBHOOK_URL: <your-slack-webhook-url>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+       post_workflow_hooks:
+         # Clean up any files left over from the run
+         - run: |
+             rm /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-infracost.json
+             rm /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-slack-message.json
```

Comment:
@hugorut Is this step working well for you with `$WORKSPACE-$REPO_REL_DIR` in the file name? It keeps failing for me as these values are not available in the post workflow hook. I removed it in this example.

Reply:
Nope, once I changed to `post_workflow_hook` this started failing - I'm bringing a PR to remove all of these from the examples as it's not needed